entrypoint.sh

```diff
@@ -122,8 +134,8 @@ wget -O $BADGE_FILE https://raw.githubusercontent.com/SAP/fosstars-rating-core-a
 git add $BADGE_FILE
 
 # Commit the report and the badge
-git config --global user.name "Fosstars"
-git config --global user.email "fosstars@users.noreply.github.com"
+git config --global user.name $GIT_USER_NAME
+git config --global user.email $GIT_USER_EMAIL
```

Comment:
```suggestion
git config --global user.name "$GIT_USER_NAME"
git config --global user.email "$GIT_USER_EMAIL"
```
Does it need to enclosed within ""

Reply:
Good catch. When there is a whitespace inside the user name it has to be enclosed with double quotes.
See https://www.git-scm.com/book/en/v2/Customizing-Git-Git-Configuration
For the email address it would not be required but I guess it makes sense to use the same syntax for better readability. I'll update the PR.

Reply:
Done